Documentation about the reading of compressed files. (#157)
r-lib/vroom: vignettes/vroom.Rmd

vroom(file)
```

> About how it work `vroom()`. After the works of this function (indexing and writes) the already uncompressed file is going to compressed again but into a temporal file (in a `tempdir()`).

Reviewer: I think you may mean temporary where you're using temporal?

Author: Yes, thank you! :)

> About how it work `vroom()`. After the works of this function (indexing and writes) the already uncompressed file is going to compressed again but into a temporal file (in a `tempdir()`).

Reviewer: I think you can collapse the first fragment with the second full-sentence (see suggestion)

Author: What you mean with that?

Reviewer: "About how it work `vroom()`" isn't actually a full sentence. I think it's implied from the surrounding context and language that you're talking about how the function works, so in my suggestion, I brought the mention of `vroom()` into that first full sentence (second, if you're counting the initial frag!)

Reviewer: So, I think this is resolved, since you accepted those edits!

> About how it work `vroom()`. After the works of this function (indexing and writes) the already uncompressed file is going to compressed again but into a temporal file (in a `tempdir()`).

Reviewer: ```suggestion
After the `vroom()` function indexes and writes the initial file, the already uncompressed file is going to compressed again but into a temporary file (in a `tempdir()`).
```

Reviewer: I think this might be missing an initial pass of decompression (cc @jimhester ?)

> The temporal file created remains until it is no longer needed, e.g. when the dataset is removed and all of the data has been materialiwed or the R session ends. After this the temporal file is automatically cleaned up. 

Reviewer: ```suggestion
The temporary file created remains until it is no longer needed, e.g. when the dataset is removed and all of the data has been materialised, or when the R session ends. After this, the temporary file is automatically cleaned up. 
```

> The temporal file is not going to be saved by default. In case you would like save this temporal file for future uses, see `vroom_write`.

Reviewer: ```suggestion
The temporary file is not saved by default. To save this temporary file for future uses, see `vroom_write()`.
```

Reviewer: Note: We follow function names with parens! 👍
